_documentation/user/flags/left-turn-only.md

```diff
@@ -5,4 +5,20 @@ type: Bad
 summary: Can't turn right.
 ---
 
-...
+The Left Turn Only flag causes increased difficulty in dodging, as right turns are no longer possible.
+
+## Tactics
+
+Left Turn Only prevents right turns and due to this, many tactics in avoiding shots are rendered ineffective.
+
+Countering the effects of Left Turn Only requires general map knowledge and some prediction skills.
```

Comment:
This is getting repetitive for every bad flag... Let's just get rid of this statement from them all

Reply:
The 'countering the effects' or the statement above or both?